authentik/core/api/devices.py

```diff
@@ -29,11 +30,20 @@ class DeviceSerializer(MetaNameSerializer):
     created = DateTimeField(read_only=True)
     last_updated = DateTimeField(read_only=True)
     last_used = DateTimeField(read_only=True, allow_null=True)
+    extra_description = SerializerMethodField()
 
     def get_type(self, instance: Device) -> str:
         """Get type of device"""
         return instance._meta.label
 
+    def get_extra_description(self, instance: Device) -> str:
+        """Get extra description"""
+        if isinstance(instance, WebAuthnDevice):
+            device_type = WebAuthnDeviceType.objects.filter(aaguid=instance.aaguid).first()
```

Comment:
WebAuthn device has a FK to the type via `device_type` (also we should return the device type and the raw AAGUID from instance.aaguid)